### ISSUE TITLE
Getting started

### ISSUE THREAD

**Issue by culmat** [Fimagena/raw2dng#24]
A little documentation on how to get started and what to expect ( man page / screen shots / feature list ) would be great.

**Comment by L1ghtmann** [comment]
For anyone curious, if you run the executable w/o any args it'll print the 'help' page. 

That being said, I do agree that this info would be useful to have in the README and/or associated with an 'h' flag.

**Comment by befocken** [comment]
I agree that this would be helpful. E.g. some concrete installation/building instructions. I found this to work for Ubuntu 20.04:

```sh
# install dependencies
sudo apt-get install -y cmake gcc g++ libexiv2-dev libraw-dev libexpat-dev libjpeg-dev zlib1g-dev

# build
mkdir build && cd build && cmake ../ && make

# install binary
sudo make install
```

**Comment by trasferetti** [comment]
Batch converting:
just `cd` to your raw files (.CR2 in this example) folder and then:
`ls *.CR2 | xargs -I{} raw2dng {} -o {}.dng  `
